MIP7/MIP7c3-Subproposals/MIP7c3-SP4.md

```diff
@@ -0,0 +1,39 @@
+# MIP7c3-SP4: Subproposal Template for Smart Contracts Domain Team Onboarding
+
+## Preamble
+```
+MIP7c3-SP#: 4
+Author(s): Wil Barnes (@wilbarnes), Lucas Manuel (@lucas-manuel), 
+  Brian McMichael (@BrianMcMakerDAO), Christopher Mooney (@cmooney)
+Contributors: n/a
+Status: Formal submission
+Date Applied: 2020-08-09
+Date Ratified: <yyyy-mm-dd>
+---
+Domain Role: Smart Contract Domain Team
+Proposed Applicant(s): Wil Barnes, Lucas Manuel, Brian McMichael, Christopher Mooney
+```
+## Application Form
+
+### Domain Team Introduction
+
+-  The Smart Contract Domain team has been working on implementing updates to the protocol, onboarding new collateral types, and writing spells to adjust governance approved risk params.
+    
+
+### Motivation
+
+- A mandated actor is required to communicate with the community to plan, schedule, and onboard new collateral types to the protocol.
```

Comment:
Would you consider adding more detail to the motivation section and replace "mandated actor" with "mandated domain team"?